src/Transports/MassTransit.EventHubIntegration/EventHubIntegration/EventHubProducerSendTransportContext.cs

```diff
@@ -175,7 +175,7 @@ public Task Send(IPipe<ProducerContext> pipe, CancellationToken cancellationToke
         }
 
         public override string EntityName => _endpointAddress.EventHubName;
-        public override string ActivitySystem => EventHubEndpointAddress.PathPrefix;
+        public override string ActivitySystem => "azure_eventhubs";
```

Comment:
```suggestion
        public override string ActivitySystem => "eventhubs";
```

Ref https://github.com/open-telemetry/semantic-conventions/pull/572/files